engines/bastion_katello/app/assets/javascripts/bastion_katello/products/details/views/product-info.html

```diff
@@ -63,13 +63,14 @@ <h4 translate>Sync Status</h4>
       <span ng-include="'products/details/partials/sync-status.html'"></span>
 
       <dt translate>Last Sync</dt>
-      <dd translate>
-        {{ product.last_sync_words }} Ago ({{ product.sync_status.finish_time | date:'medium' }} Local Time)
+      <dd>
+        <relative-date date="product.last_sync.ended_at" />
+        (<date date="product.last_sync.finish_time" /> <translate>Local Time</translate>)
```

Comment:
I think this should just be `product.last_sync` because I am seeing `product.last_sync.finish_time` as `undefined`:

![screen shot 2017-10-12 at 11 10 13 am](https://user-images.githubusercontent.com/4116405/31503539-ef43b0fa-af3d-11e7-8a9e-4012a455fe14.png)

Perhaps you need the same logic as in `repositories-info.html`, I'm not sure what our API is up to here.